src/BasedAppManager.sol

```diff
@@ -293,8 +303,9 @@ contract BasedAppManager is Initializable, OwnableUpgradeable, UUPSUpgradeable,
     // ***********************
 
     /// @notice Function to create a new Strategy
+    /// @param metadataURI The metadata URI of the bApp, which is a link (e.g., http://example.com)
```

Comment:
very minor thing, this says "metadata URI of the bApp", it should be the strategy instead